moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -1110,38 +1110,44 @@ bool PlanningSceneMonitor::getShapeTransformCache(const std::string& target_fram
                          std::vector<std::pair<occupancy_map_monitor::ShapeHandle, std::size_t>>>& link_shape_handle :
          link_shape_handles_)
     {
-      tf_buffer_->canTransform(target_frame, link_shape_handle.first->getName(), target_time,
-                               shape_transform_cache_lookup_wait_time_);
-      Eigen::Isometry3d ttr = tf2::transformToEigen(
-          tf_buffer_->lookupTransform(target_frame, link_shape_handle.first->getName(), target_time));
-      for (std::size_t j = 0; j < link_shape_handle.second.size(); ++j)
-        cache[link_shape_handle.second[j].first] =
-            ttr * link_shape_handle.first->getCollisionOriginTransforms()[link_shape_handle.second[j].second];
+      if (tf_buffer_->canTransform(target_frame, link_shape_handle.first->getName(), target_time,
+                                   shape_transform_cache_lookup_wait_time_))
+      {
+        Eigen::Isometry3d ttr = tf2::transformToEigen(
```

Comment:
any idea what `ttr` stands for? It would be nice to expand that variable name.

Reply:
I don't know what it stands for